test/src/test/java/org/corfudb/infrastructure/SequencerServerTest.java

```diff
@@ -199,23 +214,25 @@ public void SequencerWillResetTails() throws Exception {
         UUID streamB = UUID.nameUUIDFromBytes("streamB".getBytes());
         UUID streamC = UUID.nameUUIDFromBytes("streamC".getBytes());
 
-        CompletableFuture<TokenResponse> future = sendRequest(new CorfuPayloadMsg<>(CorfuMsgType.TOKEN_REQ,
-                new TokenRequest(1L, Collections.singletonList(streamA))));
+        CompletableFuture<TokenResponse> future = sendRequest(
+                getTokenRequestMsg(1L, Collections.singletonList(streamA)),
+                false, false);
         long tailA = future.join().getToken().getSequence();
 
-        future = sendRequest(new CorfuPayloadMsg<>(CorfuMsgType.TOKEN_REQ,
-                new TokenRequest(1L, Collections.singletonList(streamB))));
+        future = sendRequest(
+                getTokenRequestMsg(1L, Collections.singletonList(streamB)),
+                false, false);
         long tailB = future.join().getToken().getSequence();
 
-        future = sendRequest(new CorfuPayloadMsg<>(CorfuMsgType.TOKEN_REQ,
-                new TokenRequest(1L, Collections.singletonList(streamC))));
-        future = sendRequest(new CorfuPayloadMsg<>(CorfuMsgType.TOKEN_REQ,
-                new TokenRequest(1L, Collections.singletonList(streamC))));
+        future = sendRequest(
+                getTokenRequestMsg(1L, Collections.singletonList(streamC)),
+                false, false);
```

Comment:
Just checked that sending two requests won't change the verification logic later, so I'd prefer to keep one request since  having two identical sendRequest blocks here looks a little weird.